e2e-tests/commands/insurance/assert-companies-house-summary-list.js

```diff
@@ -0,0 +1,56 @@
+import { format } from 'date-fns';
+import { summaryList } from '../../pages/shared';
+import { COMPANIES_HOUSE_NUMBER, DATE_FORMAT } from '../../constants';
+import { INSURANCE_FIELD_IDS } from '../../constants/field-ids/insurance';
+import { FIELDS } from '../../content-strings';
+import mockCompanies from '../../fixtures/companies';
+
+const {
+  COMPANIES_HOUSE: {
+    COMPANY_ADDRESS, COMPANY_NUMBER, COMPANY_NAME, COMPANY_INCORPORATED, COMPANY_SIC, INDUSTRY_SECTOR_NAMES,
+  },
+} = INSURANCE_FIELD_IDS;
+
+const mockCompany = mockCompanies[COMPANIES_HOUSE_NUMBER];
+
+/**
+ * assertCompaniesHouseSummaryList
+ * Object structured tests for a Companies House summary list.
+ */
+const assertCompaniesHouseSummaryList = {
+  number: () => {
+    cy.checkText(summaryList.field(COMPANY_NUMBER).key(), FIELDS[COMPANY_NUMBER].SUMMARY.TITLE);
+
+    cy.checkText(summaryList.field(COMPANY_NUMBER).value(), COMPANIES_HOUSE_NUMBER);
```

Comment:
should we get this from the mockCompany instead? incase we change the number in mockCompany

Reply:
Good shout, updated